docs/Installation.md

```diff
@@ -6,13 +6,14 @@ Welcome to the Kunstmaan Bundles Standard Edition - a fully-functional CMS (cont
 
 This document contains information on how to download, install, and start using the Kunstmaan Bundles CMS.
 
-We assume you have read the official Symfony documentation and know the basics of creating Symfony applications.
+This guide assumes that you have read the [official Symfony documentation][2] and know the basics of creating
+Symfony applications.
 
 
-1) Minimal requirements
+1) Minimal Requirements
```

Comment:
The title case standard we use at the symfony docs: capitialize all words, except from closed-class words (closed-class words are words can't be extended with other words)